### ISSUE TITLE
My comment was deleted as well.

### ISSUE THREAD

**Issue by thomashan** [dreamwhite/Morgonaut-is-a-clown#1]
I left a comment on one of her videos (https://www.youtube.com/watch?v=oCUJVwNuaXA) and it was deleted almost immediately (within 10 seconds of posting the comment).

I don't think it's worthwhile arguing let alone talking to someone who thinks they're
* god of IT
* doesn't take critical feedback well
* doesn't give credit where it's due

I'd be very surprised if she doesn't use opencore for her (his) setup.

**Comment by dreamwhite** [comment]
yep, i'm too curious of what bootloader she uses since smc isn't native on our hackintoshes...

**Comment by thomashan** [comment]
(S)he should rename the domain name to `www.morgonaut.clown`.
I think (s)he is either a narcists or a sociopath maybe even both.

**Comment by Core-i99** [comment]
> 

(S)he just uses OpenCore in proxmox. I've seen this in her video and when helping one of her clients.

(It was one of her clients which decided she doesn't teach anyone anything,  doesn't explain anything. But the client wanted to actually learn something, and even called hear a witch.)

The only hypervisor which can boot macos without a hackintosh bootloader is esxi which actually has its own mini bootloader for macOS.

**Comment by iamyounix** [comment]
Stupid Maggot-naut
